### PR TITLE
Fixed cream being made from empty milk/soymilk bottles

### LINKED ISSUE #6762: The cream recipe does not require milk, just milk bottles.
### Labels

[BUG]

### Description

You can insert empty milk container into a food processor and it will output cream, this is really weird.

### Steps to reproduce

1. Empty a milk container by any means
2. Insert the milk container into the chef's food processor thing.
3. Get cream from an empty bottle. This shouldn't happen

### Expected Behavior

Not to get cream from glass.

### Additional Information & Screenshots

My guess is that the food processor only detects if you inserted the milk bottle object with no other criteria.

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
Fixes #6762

## About the PR 
Cream now requires at least 10 units of milk/soymilk present in the bottle.
I'm not aware of any other cooking recipes that have this issue, let me know if there are any

## Why's this needed? 
Cream should not be made from ground up glass

### CODE REVIEW

**Review by Gannets — APPROVED** [review]
Ah you solved this quicker than me. Nice.

File: code/obj/submachine/cooking.dm [hunk 1]
```diff
@@ -927,11 +928,14 @@ table#cooktime a#start {
 					F.reagents.add_reagent("sugar", 20)
 					qdel( P )
 				if (/obj/item/reagent_containers/food/drinks/milk)
-					new/obj/item/reagent_containers/food/snacks/condiment/cream(src.loc)
-					qdel( P )
+					if (P.reagents.has_reagent("milk") && P.reagents.get_reagent_amount("milk") >= MIN_FLUID_INGREDIENT_LEVEL)
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:931]: you don't need to test if it has the reagent

File: code/obj/submachine/cooking.dm [hunk 2]
```diff
@@ -927,11 +928,14 @@ table#cooktime a#start {
...
+						new/obj/item/reagent_containers/food/snacks/condiment/cream(src.loc)
+						qdel( P )
 				if (/obj/item/reagent_containers/food/drinks/milk/soy)
-					new/obj/item/reagent_containers/food/snacks/condiment/cream(src.loc)
-					qdel( P )
+					//so soy milk is just milk it seems, veganism is a lie
+					if (P.reagents.has_reagent("milk") && P.reagents.get_reagent_amount("milk") >= MIN_FLUID_INGREDIENT_LEVEL)
```
Comment (ZeWaka) [inline code/obj/submachine/cooking.dm:936]: you don't need to test if it has the reagent

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/submachine/cooking.dm (hunk 1 above)
Comment (TobleroneSwordfish) [reply code/obj/submachine/cooking.dm]: Gotcha, have removed it

**Pull request merged**